api/app/controllers/transaction.controller.ts

```diff
@@ -166,3 +166,48 @@ export async function findAll(req: Request, res: Response) {
       });
     });
 }
+
+export const getAllTransactions = async (req: Request, res: Response) => {
+  const page: number = Number(req.query.page) || 1;
```

Comment:
let's keep page and limit as config

Reply:
resolved here -> https://github.com/bitcointranscripts/transcription-review-backend/pull/208/commits/a0e3d50262c09755bdf01fb3bd068342d3e24f14